magma/lib/magma/query/md5_set.rb

```diff
@@ -4,6 +4,6 @@ class Md5Set < MetisMetadata
   def [](file_path)
     super
 
-    @requested_files[file_path][:file_hash]
+    @requested_files[file_path] ? @requested_files[file_path][:file_hash] : nil
```

Comment:
This wasn't when you queried for multiple file-related attributes, i.e.

```
{"project_name": "ipi", "query":["patient", ["::or", ["::has", "flojo_file_raw"]], "::all", ["updated_at", ["flojo_file_raw", "::md5"], ["flojo_file_processed", "::md5"]]]}
```

Because one of the file paths might be `nil` for any given record.